### PR TITLE
Feature/database schema

### PR DESCRIPTION
**Pull request by alexangione419**
# Checklist

- [x] I have performed a self-review of my code
- [x] I have reached out to another developer to review my code
- [x] New and existing unit tests pass locally with my changes
- [x] All Lint and CI checks are passing

### CODE REVIEW

File: backend/internal/storage/postgres/db.go [hunk 1]
```diff
@@ -4,25 +4,40 @@ import (
...
+  fmt.Println("Successfully connected to the database!")
+	return &DB{connPool: connPool}, nil
 }
 
+// Closes a connection pool, pings to confirm closure
 func (db *DB) Close(ctx context.Context) error {
-	return db.conn.Close(ctx)
+	db.connPool.Close()
+
+  err := db.connPool.Ping(ctx)
```
Comment (CamPlume1) [inline backend/internal/storage/postgres/db.go:33]: Given Ping and Close don't actually give the user any info besides success/failure, IMO we should remove as an argument and just use the background context in these methods

File: backend/internal/storage/postgres/khouryclassroomdb.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+	"github.com/CamPlume1/khoury-classroom/internal/models"
+	"github.com/jackc/pgx/v5"
+)
+
+func (db *DB) GetUsers(ctx context.Context) ([]models.User, error) {
+  
+  rows, err := db.connPool.Query(ctx, "SELECT * FROM users")
+  if err != nil {
+    return []models.User{}, err
```
Comment (CamPlume1) [inline backend/internal/storage/postgres/khouryclassroomdb.go:14]: Return nil instead of empty slice in error case

Comment (CamPlume1) [reply backend/internal/storage/postgres/khouryclassroomdb.go:14]: Just slightly better practice. Learned this myself yesterday.

**Review by CamPlume1 — CHANGES_REQUESTED** [review]
Looks awesome, super small changes

File: backend/internal/storage/postgres/db.go (hunk 1 above)
Comment (alexangione419) [reply backend/internal/storage/postgres/db.go:33]: Gotcha, to be clear this is implies we just close and return nothing?

**Review by CamPlume1 — APPROVED** [review]
LGTM

**Pull request merged**